core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java

```diff
@@ -178,10 +178,8 @@ public GroupByNumericArrayIterator(
       this.statsCollector = tsdb.getStatsCollector();
 
       if (this.result.isSourceProcessInParallel()) {
-        logger.debug("Accumulate in parallel, source size {}", sources.size());
```

Comment:
Can we keep this debug log, please? Helps during debugging

Reply:
How bout trace?

Reply:
We have a deal!